engines/registration_sharing/app/controllers/registration_sharing/rmt_to_rmt_controller.rb

```diff
@@ -34,7 +34,7 @@ def destroy
     protected
 
     def system_params
-      params.permit(:login, :password, :hostname, :proxy_byos, :system_token, :registered_at, :created_at, :last_seen_at)
```

Comment:
@jesusbv @felixsch How did the registration sharing even work without syncing the instance data? Or is it optional because the systems will get identified by login&password?

Reply:
This is because, instance_data was before in `hw_infos` with its specific handling. Now it is part of the system and can be simply added to the params